test/helpers/cilium.go

```diff
@@ -733,7 +733,7 @@ func (s *SSHMeta) DumpCiliumCommandOutput() {
 	// No need to create file for bugtool because it creates an archive of files
 	// for us.
 	res := s.ExecWithSudo(
-		fmt.Sprintf("%s -t %s", CiliumBugtool, filepath.Join(BasePath, testPath)),
+		fmt.Sprintf("%s -t \" %s \"", CiliumBugtool, filepath.Join(BasePath, testPath)),
```

Comment:
If you can change to 
```
fmt.Sprintf(`%s -t " %s "`, CiliumBugtool, filepath.Join(BasePath, testPath)),
```
Is a bit easy to read ;-)

Reply:
will make change.